dev-cpp/gtest/gtest-9999.ebuild

```diff
@@ -46,6 +46,7 @@ multilib_src_configure() {
 	local mycmakeargs=(
 		-DBUILD_GMOCK=ON
 		-DINSTALL_GTEST=ON
+		-DBUILD_SHARED_LIBS=ON
```

Comment:
This is redundant.
This has been recently dropped in https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=8341c97fd2aeccd008c55d4a88e7af48026e3f69 because BUILD_SHARED_LIBS=ON is set internally by cmake-utils.eclass for these ebuilds which use EAPI >= 7:
```
        if [[ ${EAPI} != [56] ]]; then
                cat >> "${common_config}" <<- _EOF_ || die
                        SET (CMAKE_INSTALL_DOCDIR "${EPREFIX}/usr/share/doc/${PF}" CACHE PATH "")
                        SET (BUILD_SHARED_LIBS ON CACHE BOOLEAN "")
                _EOF_
        fi
```

Reply:
@Arfrever Yep.  Forgot.  Thanks.